packages/expo/package.json

```diff
@@ -84,6 +79,7 @@
     "@types/react-test-renderer": "16.9.0",
     "@types/uuid": "^3.4.7",
     "expo-module-scripts": "^2.0.0",
+    "expo-location": "*",
```

Comment:
Specify a semver string that covers the version in this repo. Star causes issues when we go to update expo-location in this repo because yarn.lock will lock star to the old version. (There is no way to say, "Always use the version defined in this repo.")